docs/src/whatsnew/latest.rst

```diff
@@ -49,9 +49,21 @@ This document explains the changes made to Iris for this release
    property :attr:`~iris.cube.Cube.mesh` which returns a
    :class:`~iris.experimental.ugrid.Mesh` if one is attached to the
    :class:`~iris.cube.Cube` via a :class:`~iris.experimental.ugrid.MeshCoord`.
-   Finally, the context manager :obj:`~iris.experimental.ugrid.PARSE_UGRID_ON_LOAD`
+
+#. `@trexfeathers`_ added support for loading unstructured mesh data from netcdf data,
+   for files using the `UGRID`_ conventions.
+   The context manager :obj:`~iris.experimental.ugrid.PARSE_UGRID_ON_LOAD`
    provides a way to load UGRID files so that :class:`~iris.cube.Cube`\ s can be
    returned with a :class:`~iris.experimental.ugrid.Mesh` attached.
+   (:pull:`4058`).
+
+#. `@pp-mo`_ added support to save cubes with meshes to netcdf files, using the
+   `UGRID`_ conventions.
+   The existing :meth:`iris.save` function now does this, when saving cubes with meshes.
+   A routine :meth:`iris.experimental.ugrid.save_mesh` allows saving
+   :class:`iris.experimental.ugrid.Mesh` objects to netcdf _without_ any associated data
+   (i.e. not attached to cubes).
+   (:pull:`4318` and :pull:`4339`).
```

Comment:
@pp-mo This renders like this...
![image](https://user-images.githubusercontent.com/2051656/135300639-94603b0a-a529-472b-8ba0-a2c2d28dd866.png)
Is that what you intended? Or were you aiming for it to be in italics? ... if so, instead use `*without*`, or `**without**` for bold